low-level/sensors/impl/centos_7/systemd_watchdog.py

```diff
@@ -925,6 +926,7 @@ def _interface_added(self, object_path, interfaces_and_properties):
         try:
             self._log_debug("Interface Added")

```

Comment:
merge these logs & reduce logging. same for line 929, 976

Reply:
Done